src/casync-http.c

```diff
@@ -32,6 +33,39 @@ typedef enum ProcessUntil {
         PROCESS_UNTIL_FINISHED,
 } ProcessUntil;
 
+static CURLcode robust_curl_easy_perform(CURL *curl) {
+        useconds_t sleep_time = 100000;
```

Comment:
hmm, so far we used `uint64_t` for useconds... I wasn't aware useconds_t exists at all, and I figure we should actually use it more...

hmm, could you clarify this is 100ms somehow. either with a comment saying so, or by writing the number as `100*1000`, it's otherwise so hard to read so many zeroes...

Reply:
I found `useconds_t` in `usleep(3)`, didn't know about it either. But it's true that it's not used anywhere else in casync's source, so changing to `uint64_t`.

Also clarify the 100ms by using `100*1000`